src/main/java/com/epam/java/pdp/figures/Line.java

```diff
@@ -12,7 +12,7 @@
         this.end = endPoint;
     }
 
-    public double lineLength(Point start, Point end) {
+    public double lineLength() {
         double length = Math.sqrt((end.x - start.x) * (end.x - start.x) + (start.y - end.y) * (start.y - end.y));
```

Comment:
Simplify (extract local variables and use them in more complex expressions)

Reply:
hmm, I cannot get what to do here